packages/kit/src/runtime/server/index.js

```diff
@@ -250,7 +250,7 @@ export async function respond(request, options, state) {
 								try {
 									if (error) return;
 
-									const node = n ? await options.manifest._.nodes[n]() : undefined;
+									const node = n !== undefined ? await options.manifest._.nodes[n]() : undefined;
```

Comment:
```suggestion
									// == because it could be undefined (in dev) or null (in build, because of JSON.stringify)
									const node = n == undefined ? n : await options.manifest._.nodes[n]();
```